README.md

```diff
@@ -257,6 +260,26 @@ and `.dsc` files, do this:
 
 Note that we no longer actively try to get it into the Debian repository.
 
+Building the Windows Installer
+---------------------------
+
+You will need a Debian-ish operating system, and the following packages:
+
+```
+apt-get install build-essentials gcc-mingw-w64-i686 libgcrypt-mingw-w64-dev \
```

Comment:
For whatever reason I assumed that building 32-bit-only would be more reliable.

Should I create two installers, then?  One for 32-bits, one for 64?  After all, it's only double the time to create the installers, and double the space to store them, but not double the effort.  And useful for all those who have arrived in the 21st century and don't use 32-bit hardware for windows anymore :D